File: app/src/main/java/com/example/applligent/nagoriengineering/view/reminders/ShowReminders.java
package com.example.applligent.nagoriengineering.view.reminders;

import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.example.applligent.nagoriengineering.R;
import com.example.applligent.nagoriengineering.databinding.ActivityShowRemindersBinding;

public class ShowReminders extends AppCompatActivity {
    ActivityShowRemindersBinding showReminders;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle("Reminders");
        showReminders = DataBindingUtil.setContentView(this, R.layout.activity_show_reminders);
        showReminders.setReminBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(ShowReminders.this, SetReminderActivity.class));
            }
        });
        showReminders.showReminBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(ShowReminders.this, RemindersActivity.class));
            }
        });

    }
}

File: app/src/main/java/com/example/applligent/nagoriengineering/view/reminders/ShowReminders.kt
package com.example.applligent.nagoriengineering.view.reminders

import android.content.Intent

import android.os.Bundle
import android.support.v7.app.AppCompatActivity


import com.example.applligent.nagoriengineering.R

import kotlinx.android.synthetic.main.activity_show_reminders.*

class ShowReminders : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        title = "Reminders"
        setContentView(R.layout.activity_show_reminders)
        setReminBtn.setOnClickListener { startActivity(Intent(this@ShowReminders, SetReminderActivity::class.java)) }
        showReminBtn.setOnClickListener { startActivity(Intent(this@ShowReminders, RemindersActivity::class.java)) }

    }
}
